nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -176,6 +176,8 @@ class LhotseDataLoadingConfig:
     # Enables iteration of NeMo non-tarred manifests that don't have a "sampling_rate" key without performing any I/O.
     # Note that this will not allow actual dataloading; it's only for manifest iteration as Lhotse objects.
     metadata_only: bool = False
+    # Forces to ignore the "_skipme" key in data entry resulting into not skipping loading of entries with "_skipme" set to 1.
+    ignore_skipme: bool = False
```

Comment:
Prefer better naming here. double negation -> easy to confuse. 

`drop_skipme`? -> if true drops the samples for samples if _skipme key in them set to true.